docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_SectionChanger.htm

```diff
@@ -198,7 +198,7 @@ <h3>Summary</h3>
 				</tr>
 				<tr>
 					<td>
-						<pre class="intable prettyprint"><a href="#method-getActiveSectionIndex">getActiveSectionIndex</a>() </pre>
+						<pre class="intable prettyprint"><a href="#method-getActiveSectionIndex">number getActiveSectionIndex</a>() </pre>
```

Comment:
Just a question. 
Is it OK to use "number", not "Number"?

In other pages, the return types start with capital letter, while in some pages start with small letter.

Reply:
I just rewrite this type from "return value" table, so I left it as it is to keep consistency of naming in this file.

Reply:
Thanks for the answer @dorlinski.

@TomaszLukawskiSam If you think it is good enough to merge this PR, would you please re-approve this PR?